<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMatrixCnd</class>
 <widget class="QMainWindow" name="WndMatrixCnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WndMatrixCnd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="WndIrPreview" name="id_ir_frame" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>611</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="WndMatrixAlarmList" name="id_tablewidget_matrix_cnd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="id_ptn_add_matrix">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>620</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_ptn_del_martix">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_ptn_add_matrix_cnd">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>31</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_ptn_del_matrix_cnd">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>31</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_lab_martix_cnd">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>410</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WndMatrixAlarmList</class>
   <extends>QTableWidget</extends>
   <header>wndmatrixalarmlist.h</header>
  </customwidget>
  <customwidget>
   <class>WndIrPreview</class>
   <extends>QWidget</extends>
   <header>wndirpreview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
